From 85888e33e423042369c9ea693dd12167f314e1a3 Mon Sep 17 00:00:00 2001
From: Michel Thebeau <michel.thebeau@windriver.com>
Date: Fri, 26 Apr 2013 14:55:38 -0400
Subject: [PATCH 38/39] vxlan: restore unnecessary parameter for
 hlist_for_each_entry

The extra parameter was dropped upstream, but we don't have that yet, so
put the parameter back in.

Signed-off-by: Michel Thebeau <michel.thebeau@windriver.com>
---
 drivers/net/vxlan.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/vxlan.c b/drivers/net/vxlan.c
index 7c82160..ac1ba91 100644
--- a/drivers/net/vxlan.c
+++ b/drivers/net/vxlan.c
@@ -1506,10 +1506,11 @@ static __net_exit void vxlan_exit_net(struct net *net)
 	struct vxlan_net *vn = net_generic(net, vxlan_net_id);
 	struct vxlan_dev *vxlan;
 	unsigned h;
+	struct hlist_node *n;
 
 	rtnl_lock();
 	for (h = 0; h < VNI_HASH_SIZE; ++h)
-		hlist_for_each_entry(vxlan, &vn->vni_list[h], hlist)
+		hlist_for_each_entry(vxlan, n, &vn->vni_list[h], hlist)
 			dev_close(vxlan->dev);
 	rtnl_unlock();
 
-- 
1.7.9.7

